recipes/python-gmatnonlinearelastic/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "python-gmatnonlinearelastic" %}
+{% set version = "0.0.4" %}
+{% set sha256 = "faaa3f1e8fa8d8b121e4164e63c1fa0c6b2d109ee8f1be90f2d399ec8df42f3a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tdegeus/GMatNonLinearElastic/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  script:
+    - python setup.py build
```

Comment:
Have you tried using pip?

Reply:
@chrisburr I successfully switched to pip, thanks!